docs/tutorials/matrix.rst

```diff
@@ -81,8 +81,9 @@ and there to purely understand the API):
 
     scip.addMatrixCons(x + y <= z)
     scip.addMatrixCons(exp(x) + sin(sqrt(y)) == z + y)
-    scip.addMatrixCons(y <= x @ y <= x)
-    scip.addMatrixCons(x.sum() <= 2)
+    scip.addMatrixCons(y <= x @ y)
+    scip.addMatrixCons(x @ y <= x)
+    scip.addCons(x.sum() <= 2) # Matrix variables can also appear in standard constraints, if the result expression as type Expr
```

Comment:
This comment is incorrect? Or at least I don't understand it. I will remove it.

Reply:
Ahhhhhhh. Gotcha. Will leave it as is.

Reply:
*is type Expr